indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -378,6 +378,9 @@
           :science-keywords science-keywords-field-mapping
           :science-keywords.humanized science-keywords-field-mapping
 
+          :granule-data-format                  (m/stored m/string-field-mapping)
```

Comment:
Shouldn't we have a `:granule-data-format.lower-case` field to support case insensitive search?

Reply:
This issue is only for humanized granule-data-format, we could not index `:granule-data-format` at all, but I guess we would want to search/retrieve it eventually.

Reply:
Fair point. I added the `.lowercase` field anyway